fashion-example-query/flows/index.yml

```diff
@@ -0,0 +1,25 @@
+!Flow
+version: '1'
+with:
+  compress_hwm: 1024
+pods:
+  - name: encode
+    uses: pods/encode.yml
+    parallel: $JINA_PARALLEL
```

Comment:
make sure you consistently use `shards` (`shards: $JINA_SHARDS_ENCOER` etc.)

Reply:
By this you mean that in general we will use `parallel` as `JINA_SHARDS_ENCOER`?

Reply:
I meant:

1. Let's try to always use `shards` (and `parallel` will be reserved for other architecture options - parallel pods etc.)
2. Name the env. vars. with `JINA_SHARDS_COMPONENT` where `COMPONENT` is the specific component for which you're setting that number of shards.